### ISSUE TITLE
Warning message after Mandelbulber renders an OpenCL image 

### ISSUE THREAD

**Issue by TasMania17** [buddhi1980/mandelbulber2#965]
I don't think this effects OpenCL rendering on Mandelbulber compiled on Debian 12 stable.
`'ptxas application ptx input, line 9; fatal   : Unsupported .version 8.4; current version is '8.0''`

![MDBB-error](https://github.com/buddhi1980/mandelbulber2/assets/32868462/167394f1-093f-4bf5-b508-a8df33edf96e)

I found the issue is cause by the CUDA JIT compiler
libnvidia-ptxjitcompiler1 525.147.05-7~deb12u1
It installs this file:
/usr/lib/x86_64-linux-gnu/nvidia/current/libnvidia-ptxjitcompiler.so.525.147.05

Debian Experimental has this file at version 8.3 so this appears not to be installable on Debian.

I want to avoid installing the whole CUDA toolkit because it's big and has to synchronise with the nvidia driver.

**Comment by buddhi1980** [comment]
I run Mandelbulber on 'testing' version of Debian and everything is fine. Actual driver version is 535.161.08

**Comment by TasMania17** [comment]
Thanks buddhi, I fixed this issue.
I was going to upgrade to testing eventually. After I upgraded and re-compiled Mandelbulber I was still getting the same error message. From `.mandelbulber` folder I moved `openclCache` directory to Rubbish. No more pop-ups but if I restore that dir they start again.  :) The directory is re-made automatically. It must have been corrupted.

I found MDBB compiles and runs on Debian stable. Also the `prepare_for_dev_debian_testing` and `prepare_for_dev_ubuntu` scripts work exactly the same on Debian testing and stable and both make Mandelbulber .
